backend/internal/pod_data/measurement.go

```diff
@@ -87,12 +87,16 @@ func getEnumMeasurement(adeMeas adj.Measurement) EnumMeasurement {
 		Id:      adeMeas.Id,
 		Name:    adeMeas.Name,
 		Type:    EnumType,
-		Options: getEnumMembers(adeMeas.EnumValues),
+		Options: getEnumMembers(adeMeas.Type),
```

Comment:
diría que esto está mal, `getEnumMembers()` recibe un string de varios valores, dudo que `adeMeas.Type` sea un string de varios valores separados por coma, pruebalo a ver que pasa.

Reply:
It receives an individual string

Reply:
The type includes "Type(member1,member2,member3...)"

Reply:
Ok, I approve then